### ISSUE TITLE
question about background services

### ISSUE THREAD

**Issue by Cersin** [siprix/FlutterPlugin#1]
Hi,
Can you receive calls with this app when it is turned off with this flutter plugin? Can we integrate it with pushes?

**Comment by siprix** [comment]
Yes, you can add this plugin into own project, add firebase_messaging, and implement '_firebaseMessagingBackgroundHandler' which starts activity and becomes ready to receive calls.

**Comment by siprix** [comment]
Where you able to configure push notifications in the example app?
Do you need some help or reference implementation for testing?
Can you plz provide more details about app, which is required to implement?27.11.2024 09:24, Szymon Wójcik ***@***.***>:> Hi,> Can you receive calls with this app when it is turned off with this flutter plugin? Can we integrate it with pushes?> --> Reply to this email directly or view it on GitHub:> https://github.com/siprix/FlutterPlugin/issues/1> You are receiving this because you are subscribed to this thread.> Message ID:
